src/environments/environment.ts

```diff
@@ -5,6 +5,6 @@
 export const environment = {
   production: false,
   profileUrl:
-    'https://landville-backend-web-api.herokuapp.com/api/v1/auth/profile/',
-  api_url: 'https://landville-backend-web-api.herokuapp.com/api/v1',
+    'http://127.0.0.1:8000/api/v1/auth/profile/',
+  api_url: 'http://127.0.0.1:8000/api/v1',
```

Comment:
Don't forget to change this back to the production URLs

Reply:
Thank you, @Tevinthuku I won't forget.